samples/cellular/nrf_cloud_mqtt_multi_service/prj.conf

```diff
@@ -65,6 +65,7 @@ CONFIG_AT_HOST_LIBRARY=y
 
 # nRF Cloud
 CONFIG_NRF_CLOUD_MQTT=y # This also enables FOTA by implicitly setting CONFIG_NRF_CLOUD_FOTA=y
+CONFIG_NRF_CLOUD_FOTA=y
```

Comment:
TODO:  remove, but update docs to indicate it can now be disabled (just like the other services this sample uses can already be disabled)

Reply:
Was this added to the docs? Didn't notice, at least.

Reply:
Done.